components/RoomNav/index.js

```diff
@@ -0,0 +1,78 @@
+import TalkList from "../TalkList";
+import TalkItem from "../TalkItem";
+import TalkInfoWrapper from "../TalkInfoWrapper";
+import FooterNav from "../FooterNav";
+import StyledLink from "../StyledLink";
+import StyledRoom from "../StyledRoom";
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function RoomNav({ talks }) {
+  const rooms = ["Great Hall", "Liberty Hall", "Metro West"];
+  const [roomIndex, setRoomIndex] = useState(0);
+
+  const talksInCurrentRoom = talks
+    .filter((talk) => talk.room === rooms[roomIndex])
+    .sort((a, b) => {
+      let timeA = a.time;
+      let timeB = b.time;
+
+      return timeA < timeB ? -1 : 1;
+    })
+    .sort((a, b) => {
+      let sessionA = a.session;
+      let sessionB = b.session;
+
+      return sessionA < sessionB ? -1 : 1;
+    });
+
+  function handleNextRoomButton() {
+    if (roomIndex < rooms.length - 1) {
+      setRoomIndex((prev) => prev + 1);
+    }
+    if (roomIndex === 2) {
+      setRoomIndex(0);
+    }
+  }
+
+  function handlePrevRoomButton() {
+    if (roomIndex < rooms.length) {
+      setRoomIndex((prev) => prev - 1);
+    }
+    if (roomIndex === 0) {
+      setRoomIndex(2);
+    }
```

Comment:
This could be also a bit more generic. I would suggest the following:

```suggestion
if (roomIndex === 0) {
  roomIndex = rooms.length-1;
} else {
  roomIndex = roomIndex - 1;
}
```

Reply:
same as above

Reply:
Also updated :)